### PR TITLE
bugfix: fix file.conf read failed after package

### LINKED ISSUE #6888: Read file.conf in wrong path after package seata server
- [x] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
When I use idea to read file.conf in resources. it is ok.
f4a5e259cb931496979e16d5ce4f5c22

When I package server, it can not read file.conf in right path.
4b3dd01a1638c4eab39e5cc9fc706d25

### Ⅱ. Describe what happened

I think there are some errors after package, then change the result in here .
image

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

Minimal yet complete reproducer code (or URL to code):

### Ⅵ. Environment:

- JDK version(e.g. `java -version`):
- Seata client/server version:  
- Database version:
- OS(e.g. `uname -a`):
- Others:

### PR DESCRIPTION
**Pull request by xjlgod**
- [x] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
fix https://github.com/apache/incubator-seata/issues/6888

### CODE REVIEW

File: changes/en-us/2.x.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Add changes here for all PR submitted to the 2.x branch.
 - [[#6876](https://github.com/apache/incubator-seata/pull/6876)]support kingbase
 
 ### bugfix:
-
+- [[#6899](https://github.com/apache/incubator-seata/pull/6899)] fix: fix file.conf read failed after package
```
Comment (funky-eyes) [inline changes/en-us/2.x.md]:
```suggestion
- [[#6899](https://github.com/apache/incubator-seata/pull/6899)] fix file.conf read failed after package
```

File: config/seata-config-core/src/main/java/org/apache/seata/config/FileConfiguration.java [hunk 2]
```diff
@@ -16,13 +16,25 @@
  */
 package org.apache.seata.config;
 
+import org.slf4j.Logger;
```
Comment (funky-eyes) [inline config/seata-config-core/src/main/java/org/apache/seata/config/FileConfiguration.java]: 移到java包下方
Move below the java package

File: changes/en-us/2.x.md (hunk 1 above)
Comment (xjlgod) [reply changes/en-us/2.x.md]: Ok.

File: config/seata-config-core/src/main/java/org/apache/seata/config/FileConfiguration.java (hunk 2 above)
Comment (xjlgod) [reply config/seata-config-core/src/main/java/org/apache/seata/config/FileConfiguration.java]: Have done.

File: changes/zh-cn/2.x.md [hunk 3]
```diff
@@ -37,6 +37,7 @@
 - [yougecn](https://github.com/yougecn)
 
 
+- [xjlgod](https://github.com/xjlgod)
```
Comment (funky-eyes) [inline changes/zh-cn/2.x.md]:
```suggestion
- [yougecn](https://github.com/yougecn)
- [xjlgod](https://github.com/xjlgod)
```

**Review by funky-eyes — APPROVED** [review]
LGTM

**Pull request merged**